app/views/core_induction_programme/modules/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l"> <%= @course_module.title %></h1>
+
+    <div class="gem-c-govspeak govuk-govspeak ">
+      <%= @course_module.content_in_html.html_safe %>
+    </div>
+
+    <ol class="app-task-list">
+      <li>
+        <p><strong>Topic materials</strong></p>
+        <ul class="app-task-list__items">
+          <% @course_module.course_lessons.each do |lesson| %>
```

Comment:
What happens when there are no course lessons? Do we need to show a message for that?

Reply:
Probably. What happens is, quality control is annoyed that we called something a 'module' when it might not be 😂